.github/workflows/test.yaml

```diff
@@ -110,6 +125,9 @@ jobs:
         run: |
           bundle exec rake test:spatialite
           bundle exec rake test:sqlite3
+      - name: Run trilogy tests
+        if: ${{ matrix.env.AR_VERSION >= '7.0' && matrix.ruby != 'jruby' }}
```

Comment:
I'm not entirely sure where the trilogy gem stands with jruby, but I couldn't figure out a way to get it to successfully run.